docs/pages/identity-governance/entra-id/group-filters.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Entra ID Group Filters
+description: How to configure group filters for the Teleport Entra ID integration.
+sidebar_position: 5
+tags:
+ - identity-governance
+ - azure
+ - privileged-access
+---
+
+This page explains how to configure group filters in the Teleport Entra ID integration. 
```

Comment:
Since this guide is fairly short, and for a specific field of the Entra ID plugin configuration, I think it would make sense to incorporate it into a more general-purpose guide, such as the [Getting Started with the Entra ID integration guide](https://sshah-update-entra-docs-for-group-filters.d212ksyjt6y4yg.amplifyapp.com/identity-governance/entra-id/getting-started/).

I think this would be especially beneficial since users configure group filters using `tctl plugins install entraid`, a command that one of the Getting Started guide's purposes is to explain. In the current organization, a user would run the `tctl plugins install` command while following the Getting Started guide, only to realize while following this guide that they were missing a flag.